gnovm/pkg/packages/imports_test.go

```diff
@@ -1,11 +1,12 @@
-package packages
+package packages_test
 
 import (
 	"os"
 	"path/filepath"
 	"testing"
 
 	"github.com/gnolang/gno/gnovm/pkg/gnolang"
+	. "github.com/gnolang/gno/gnovm/pkg/packages"
```

Comment:
why?

Reply:
I saw that in the go standard libraries they are using dot import for the current package in external tests, thus thought it was the idiomatic go way. I can use a normal import if you prefer